test/integration/view_pursuits_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class ViewPursuitsTest < ActionDispatch::IntegrationTest
+  def create_pursuits(num)
+    num.times do |i|
+      i += 1
+      Pursuit.create(name: "Hiking the Alps #{i}", description: "Go hike the alps! #{i}", price: 1000+i) #rou, activity_id: i, location_id: i)
```

Comment:
Line is too long. [142/80]<br>Missing space after #.<br>Surrounding space missing for operator `+`.